packages/salesforcedx-visualforce-language-server/src/modes/embeddedSupport.ts

```diff
@@ -5,7 +5,7 @@
 
 'use strict';
 
-import { LanguageService, TokenType } from '@salesforce/salesforcedx-visualforce-markup-language-server';
+import { LanguageService, TokenType } from '@salesforce/salesforcedx-visualforce-markup-language-server/out/src';
```

Comment:
by adding the missing `main` prop on pjson, we can import from TS instead of the compiled code